### ISSUE TITLE
Missing path/format for .ttf files in generated css

### ISSUE THREAD

**Issue by mikestempien** [matthewgonzalez/fontplop#27]
When converting from TTF app doesn't build OTF file, yet it links to it in CSS (while not linking to TTF)

**Comment by briangonzalez** [comment]
I think the best approach is to switch it over from OTF to TTF, which I believe is more widely used anyways.

**Comment by briangonzalez** [comment]
Also, thanks for the report!

**Comment by mikestempien** [comment]
Yeah, TTFs are fine, just letting you know that the app doesn't link them properly

**Comment by briangonzalez** [comment]
10-4

I've got a PR up which addresses this issue: https://github.com/matthewgonzalez/fontplop/pull/28

**Comment by kevinmu17** [comment]
This problem still persists

**Comment by rafaelmaeuer** [comment]
Yes exported css should reference .ttf files instead of .otf, if input files are .ttf

**Comment by rafaelmaeuer** [comment]
> 10-4
> 
> I've got a PR up which addresses this issue: #28

This should be already solved with this PR, @matthewgonzalez would you mind to release the new v1.4.0 which includes this fix?
